requirements.txt

```diff
@@ -74,6 +74,7 @@ pyOpenSSL==17.3.0
 python-dateutil==2.6.1
 pytz==2017.3
 PyYAML==3.12
+requests==2.18.4
```

Comment:
this file shouldn't be directly edited, add this pin to the `requirements-top-level.txt`

Reply:
gotcha.  it looks like you already updated the requests requirement a few days ago so this is redundant anyways.